### PR TITLE
Removed default true from inProgress

### PR DESCRIPTION
**Pull request by shaior**
Test-Hint: no-check
TestDallas: test

### CODE REVIEW

**Review by natifridman — APPROVED** [review]

**Comment by natifridman** [comment]
recheck

**Comment by natifridman** [comment]
dci-pipeline-check

**Comment by shaior** [comment]
recheck

File: roles/jenkins_job_launcher/tasks/main.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@
     user: "{{ jjl_username }}"
     password: "{{ jjl_token }}"
   register: in_progress
-  until: in_progress.json.inProgress|default("true") == "false"
+  until: in_progress.json.inProgress == "false"
```
Comment (tonyskapunk) [inline roles/jenkins_job_launcher/tasks/main.yml:42]: Maybe?
```suggestion
  until:
    - in_progress.json is defined
    - in_progress.json.inProgress is defined
    - not in_progress.json.inProgress | bool
```

**Comment by shaior** [comment]
dci-pipeline

**Comment by shaior** [comment]
recheck

**Comment by shaior** [comment]
retest

**Comment by fredericlepied** [comment]
retest

**Review by natifridman — APPROVED** [review]

**Comment by tonyskapunk** [comment]
recheck

**Pull request merged**